src/fromevent.js

```diff
@@ -1,8 +1,7 @@
-let Rx = require(`rx`)
+let Rx = require(`@reactivex/rxjs`)
 
-const disposableCreate = Rx.Disposable.create
-const CompositeDisposable = Rx.CompositeDisposable
-const AnonymousObservable = Rx.AnonymousObservable
+const disposableCreate = (func) => new Rx.Subscription(func)
```

Comment:
Agreed about refactoring, things will get much smoother once we delete custom elements.